Packages/Sites/Sfi.Kateheo/Resources/Private/TypoScript/NodeTypes/Tags.ts2

```diff
@@ -0,0 +1,26 @@
+prototype(Sfi.Kateheo:Tags) {
+	layoutRootPath = 'resource://Sfi.Kateheo/Private/Layouts'
+	partialRootPath = 'resource://Sfi.Kateheo/Private/Partials'
+
+	aMenu = Menu {
+		itemCollection = ${q(site).find('[instanceof Sfi.Kateheo:Tags]').get()}
+		itemCollection.@process.sort = ${q(value).count() > 0 ? q(value).sort('title', 'ASC').get() : value}
+		attributes.class = "nav nav-list bs-docs-sidenav affix-top"
+		current.attributes.class = "active"
+	}
+
+	main = TYPO3.Neos:ContentCollection{
+		nodePath = 'main'
+	}
+	newsList = Sfi.Listable:List {
+		@context.listClass = 'news-list-view'
+		prototype(Sfi.Listable:Listable) {
+			@context.sortProperty = 'date'
```

Comment:
Yep. But you can leave it to be more explicit if you like. Also it would be great to migrate to latest version of Flowpack.Listable, but maybe not now